rest_framework/permissions.py

```diff
@@ -151,7 +151,7 @@ class DjangoModelPermissions(BasePermission):
     # Override this if you need to also provide 'view' permissions,
     # or if you want to provide custom permission codes.
     perms_map = {
-        'GET': [],
+        'GET': ['%(app_label)s.view_%(model_name)s'],
         'OPTIONS': [],
         'HEAD': [],
```

Comment:
I think the `view_*` permission check should be applied to `HEAD` requests as well.

Sure, the body isn't returned but having access to `content-length` could leak information about the page. Also, having `GET` and `HEAD` return different status codes seems weird to me.

Reply:
Agreed. We'd want the same permission on `HEAD` requests, yup.

Also the "Override this if you need to also provide 'view' permissions" part of the comment can be dropped.